ord_schema/validations.py

```diff
@@ -318,11 +257,10 @@ def validate_vessel(message):
             not message.preparation_details:
         warnings.warn('VesselPreparation custom, but no details provided',
                       ValidationError)
-    return message
 
 
 def validate_reaction_setup(message):
-    return message
+    del message  # Unused.
```

Comment:
I don't understand what the intended behavior is here. Why `del message` and not, e.g., `pass`? Same comment for the other functions below.

Reply:
pylint complains if `message` is unused; I had `pass` originally but changed to this so we could avoid changing the function signature. I could replace `message` with `_` in the signature if you prefer.

Reply:
Ah, it makes sense that pylint would complain. `del` seems like an odd placeholder still, but that's okay